app-misc/fastfetch/patches/fastfetch-2.36.0.patchset

```diff
@@ -0,0 +1,38 @@
+From c452d0ca4b622f20e523bbc02d185e7b71d79265 Mon Sep 17 00:00:00 2001
+From: Begasus <begasus@gmail.com>
+Date: Mon, 10 Feb 2025 20:06:55 +0100
+Subject: Add _GNU_SOURCE, should fix memmem error on buildmaster
+
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 426cd0e..5016ec1 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -1275,6 +1275,8 @@ elseif(DragonFly)
+     target_compile_definitions(libfastfetch PUBLIC __FreeBSD__)
+ elseif(SunOS)
+     target_compile_definitions(libfastfetch PUBLIC _GNU_SOURCE _XOPEN_SOURCE __STDC_WANT_LIB_EXT1__ _FILE_OFFSET_BITS=64 __EXTENSIONS__ _POSIX_C_SOURCE)
++elseif(HAIKU)
```

Comment:
Shouldn't be `Haiku`?

Reply:
Too much KDE porting done with this in mind. :)